src/shared/Dropdown.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, JSX, Show, createSignal } from "solid-js";
+
+import { ChevronDown } from "lucide-solid";
+import Button from "./Button";
+
+
+const Dropdown: Component<{ children: JSX.Element; label: string }> = (
+  props
+) => {
+  const [open, setOpen] = createSignal(false);
+
+  return (
+    <div class="inline-block">
+      <Button
+        onClick={() => {
+          setOpen(!open());
+        }}
+      >
+        <ChevronDown />
+        {props.label}
+      </Button>
+      <Show when={open()}>{props.children}</Show>
```

Comment:
If a dropdown is assumed to have a container with class `.dropdown-menu`, why not put `props.children` inside of here? Is there potential for a case where we might want to use this component to display non-menu content?

Reply:
if u mean changing the Show component to something like this then i agree makes things more concise, I'll push commit
```
      <Show when={open()}>
        <div class="dropdown-menu">
          {props.children}
        </div>
      </Show>
```

so example usage would be like 
```
    <Dropdown label="Preset">
      <a class="dropdown-item" href="#">
        Classic-Pygmalion-6b
      </a>
      <a class="dropdown-item" href="#">
        Classic-Pygmalion-2.7b
      </a>
      <a class="dropdown-item" href="#">
        Calibrated-Pygmalion-6b
      </a>
      <a class="dropdown-item" href="#">
        GPU-Pygmalion-6b
      </a>
      <a class="dropdown-item" href="#">
        DragonSlayer-Pygmalion-2.7b
      </a>
    </Dropdown>
```

Reply:
Hahaha yeah, sorry, that's what I meant.